cornac/eval_methods/propensity_stratified_evaluation.py

```diff
@@ -288,11 +288,11 @@ def _build_stratified_dataset(self, test_data):
                                for u, i, r in test_data], dtype=np.float64)
 
         # stratify
```

Comment:
If your objective is to have the same output as the old pandas code, you can try this:
```python
minp = min(test_props)
maxp = max(test_props)
slice = (maxp - minp) / self.n_strata
strata = np.digitize(x=test_props, bins=np.arange(minp, maxp, slice), right=True)
strata[np.argmin(strata)] = 1
```